slither/printers/summary/printer_human_summary.py

```diff
@@ -0,0 +1,102 @@
+"""
+Module printing summary of the contract
+"""
+import logging
+
+from slither.detectors.detectors import Detectors
+from slither.printers.abstractPrinter import AbstractPrinter
+from slither.utils.code_complexity import compute_cyclomatic_complexity
+from slither.utils.colors import green, red, yellow
+
+
+class PrinterHumanSummary(AbstractPrinter):
+    ARGUMENT = 'print-human-summary'
+    HELP = 'Print an human readable summary of the contracts'
+
+    @staticmethod
+    def get_summary_erc20(contract):
+        txt = ''
+        functions_name = [f.name for f in contract.functions]
+        state_variables = [v.name for v in contract.state_variables]
+
+        if 'paused' in functions_name or 'pause' in functions_name:
+            txt += "\t\t Can be paused? : {}\n".format(yellow('Yes'))
+        else:
+            txt += "\t\t Can be paused? : {}\n".format(green('No'))
+
+        if 'mint' in functions_name:
+            if not 'mintingFinished' in state_variables:
+                txt += "\t\t Minting restriction? : {}\n".format(red('None'))
+            else:
+                txt += "\t\t Minting restriction? : {}\n".format(yellow('Yes'))
+        else:
+            txt += "\t\t Minting restriction? : {}\n".format(green('No Minting'))
+
+        if 'increaseApproval' in functions_name:
+            txt += "\t\t ERC20 race condition mitigation: {}\n".format(green('Yes'))
+        else:
+            txt += "\t\t ERC20 race condition mitigation: {}\n".format(red('No'))
+
+        return txt
```

Comment:
We could also have like:
```python
def colored(condition, true='Yes', false='No'):
    return green(true) if condition else red(false)

paused = colored('pause' in functions_name)

if 'mint' in functions_name:
    mint = colored('mintingFinished' in state_variables, false='None')
else:
    mint = green('No Minting')

race_cond = colored('increaseApproval' in functions_name)

return (
    f'\t\t Can be paused? {paused}\n'
    f'\t\t Minting restriction? {mint}\n'
    f'\t\t ERC20 race condition mitigation: {race_cond}\n'
)
```

Reply:
Also, maybe we should use some more general names that `green` and `red` for colors, so it is would be possible to configure colors in one place?

we could have like `ok(msg)` and `err(msg)` or `yes` and `no` or sth like that.